src/connector.rs

```diff
@@ -388,6 +388,7 @@ impl EthConnectorContract {
     }
 
     /// Burn NEAR tokens
+    #[allow(dead_code)]
```

Comment:
If we agree on this solution then I think we should delete the function instead of allowing it to exist as dead code. If we end up needing it in the future we can always restore it from the git history.

Reply:
Done!